.github/workflows/test.yml

```diff
@@ -0,0 +1,64 @@
+# yaml-language-server: $schema=https://json.schemastore.org/github-workflow
+
+name: Tests
+
+on:
+  pull_request:
+  push:
+    branches:
+      - trunk
+
+jobs:
+  test:
+    name: Test / PHP ${{ matrix.php }}
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    timeout-minutes: 10
+
+    strategy:
+      matrix:
+        php:
+          - "8.2"
+          - "7.4"
+      fail-fast: false
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Install PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          tools: composer:v2
+          coverage: none
+          ini-file: development
+        env:
+          fail-fast: true
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Debugging
+        run: | #shell
+          php --version
+          composer --version
```

Comment:
Why do you always add these?
The output of shivammathur/setup-php contains both versions.